components/Main/Workouts/Workouts.js

```diff
@@ -27,32 +27,35 @@ const Workouts = ({ workouts, onAddWorkout, onDelete }) => {
 
   return (
     <>
-      <StyledWorkoutsHeader>
-        <h2>My-Workouts</h2>
-        <button onClick={() => toggleShow()}>
-          <Image
-            className="add-workout"
-            src="/plus-circle.svg"
-            width={32}
-            height={32}
-            alt="show input"
-          />
-        </button>
+      <StyledWorkoutsHeader
+        style={{ maxHeight: !showInput ? "50px" : "100px" }}
+      >
+        <div className="workoutsHeaderContainer">
+          <h2>My-Workouts</h2>
+          <button className="add-workout" onClick={toggleShow}>
+            <Image
+              src="/plus-circle.svg"
+              width={32}
+              height={32}
+              alt="show input"
+            />
+          </button>
+        </div>
+        <InputWorkout onSubmit={handleSubmit} />
       </StyledWorkoutsHeader>
-      {showInput ? <InputWorkout onSubmit={handleSubmit} /> : null}
 
       <StyledWorkouts>
         <ul>
           {workouts.map((workout) => {
             return (
-              <Link href={"/workouts/" + workout.id} key={workout.id}>
-                <li>
-                  <div className="workoutContainer">
+              <Link href={"/workouts/" + workout.id}>
+                <li key={workout.id}>
+                  <div className="workoutContainer" key={workout.id}>
```

Comment:
```suggestion
                <li key={workout.id}>
                  <section className="workoutContainer">
```
den key vergibst du hier doppelt

Reply:
der Link drüber benötigt den Key, weil direktes child von UL sonst meckert er :)